modules/nf-core/ngscheckmate/patterngenerator/main.nf

```diff
@@ -25,7 +25,9 @@ process NGSCHECKMATE_PATTERNGENERATOR {
     if ("$fasta" == "${prefix}.fasta") error "makesnvpattern.pl generates a fasta file with the same name as the input fasta, use \"task.ext.prefix\" to disambiguate!"
 
     """
-    makesnvpattern.pl ${bed} ${fasta} ${bowtie_index}/${fasta.getBaseName()} . ${prefix}
+    INDEX=`find -L ./ -name "*.3.ebwt" | sed 's/\\.3.ebwt\$//'`
```

Comment:
Can you remind me what the list of files here is? I recall another instance where the pattern match wasn't unique because it also matched the `rev` something `ebwt`.

Also could we update it to the modern bash syntax for command substitution?` `\$( ... )` rather than backticks?

Reply:
There are two rev files, 4 ebwt files, which is why it is looking for number 3. I took this from the bowtie align module.